test/storage/recovery_test.cpp

```diff
@@ -315,7 +363,7 @@ TEST_F(RecoveryTests, DropDatabaseTest) {
   DiskLogProvider log_provider(LOG_FILE_NAME);
   RecoveryManager recovery_manager(&log_provider, common::ManagedPointer(recovery_catalog_), recovery_txn_manager_,
                                    recovery_deferred_action_manager_, common::ManagedPointer(thread_registry_),
-                                   &block_store_);
+                                   &block_store_, recovery_metric_interval_, false);
```

Comment:
Like I mentioned in another comment, create a `MetricsThread`, and pass in that `MetricsThread`'s `MetricsManager` here